recipes/mecab/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+cd mecab
+./configure --prefix=$PREFIX --with-charset=utf8
+make -j${CPU_COUNT}
+make install
```

Comment:
Is there a make check we can run here?

Reply:
There is!  Have added it to this script. 👍

Reply:
It should be before `make install`, no?

Reply:
I figured it didn't really matter as long as the script ran both, but yeah running check first does make more sense.  Have switched them.